### PR TITLE
add InkPresenter interface

### PR DESCRIPTION
**Pull request by chrisdavidmills**
#### Summary

This PR adds the Ink API's [InkPresenter](https://wicg.github.io/ink-enhancement/#ink-presenter) interface to BCD.

#### Test results and supporting details

The other features of the API were already in BCD, so I just copied the support data from them. It looks fairly accurate tho — the Chrome folks have confirmed to me that it is definitely supported in Chrome 94 and above, and I also tested it in Edge and found it to be working 

See [Enhancing Inking on the Web](https://blogs.windows.com/msedgedev/2021/08/18/enhancing-inking-on-the-web/) for more explanation and a demo.

### CODE REVIEW

File: api/InkPresenter.json [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    "InkPresenter": {
+      "__compat": {
+        "spec_url": "https://wicg.github.io/ink-enhancement/#ink-presenter",
+        "support": {
+          "chrome": {
+            "version_added": "94"
+          },
+          "chrome_android": "mirror",
+          "edge": {
+            "version_added": "93"
```
Comment (teoli2003) [inline api/InkPresenter.json:12]: This is weird. How is it that the Edge version is before the Chrome version? (Here and for the other ink).

@queengooborg Can we test this?

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: api/InkPresenter.json (hunk 1 above)
Comment (chrisdavidmills) [reply api/InkPresenter.json:12]: It is a bit strange...maybe it should just be 94.

Comment (queengooborg) [reply api/InkPresenter.json:12]: Microsoft led the development of Ink APIs so Edge getting the feature early actually does make sense.  I think the latest mdn-bcd-collector version has the IDL for this interface, so if desired, we can confirm the data through it!

(By the way, welcome back Chris, it's great to see you again!)

Comment (chrisdavidmills) [reply api/InkPresenter.json:12]: > (By the way, welcome back Chris, it's great to see you again!)

👋 ❤️ !

Nice to see you too ;-)

**Review by teoli2003 — APPROVED** [review]
I think your explanation… explain why. Thanks @queengooborg
